tests/test_telcameras_v2.py

```diff
@@ -465,3 +466,47 @@ def test_scramble_counts_with_counts_zero(self):
         self.assertIn(count_agg.count_in_scrambled, (0, 1))
         self.assertIn(count_agg.count_out_scrambled, (0, 1))
         self.assertIn(count_agg.count_scrambled, (0, 1))
+
+    def test_scramble_v2_counts_command(self):
+        baker.make(CountAggregate, _fill_optional=['count_in', 'count_out', 'count'], _quantity=1100)
+        for ca in CountAggregate.objects.all():
+            # ModelBaker doesn't take the smallinteger into account, so we adjust for it here
```

Comment:
Weet je dit zeker, want in de code lijkt het er op dat ze dat wel doen: https://github.com/model-bakers/model_bakery/blob/297745bad159de0559a779d3c3da83770082ca60/model_bakery/generators.py#L120

Reply:
Alternatief is dat je een recipe gebruikt (waarschijnlijk sowieso wat je wilt omdat je nu ook grote negatieve getallen kan krijgen):

```
count_aggregate_recipe = Recipe(
    CountAggregate,
    count_in=random.randint(0, 1000),
    count_out=random.randint(0, 1000),
    count=random.randint(0, 1000),
    count_in_scrambled=None,
    count_out_scrambled=None,
    count_scrambled=None,
)
count_aggregate_recipe.make(_quantity=1100)
```

Dan kunnen de regels 474 tot 485 weg.